distribution/sender/routes/processor-route.xml

```diff
@@ -4,48 +4,7 @@
         <from uri="direct:db-event-processor" />
 
         <log message="Processing db event: ${body}" loggingLevel="DEBUG" />
-
```

Comment:
You should not be touching this route possibly

Reply:
Yes. But this route used to contain unwanted code (the one to identify the uuid). Now we identify the uuid on debezium route.

Reply:
Which unwanted code?

Reply:
The uuid being set here is when processing a subclass table, so we need it

Reply:
But you already have the uuid on the stating table. If you look at debezium-route you will see the attached code which initialy was in the processor-route
![RemovedCode](https://user-images.githubusercontent.com/4964616/119940312-4abc1200-bf8f-11eb-89a8-4e36c2765ccf.PNG)

Reply:
But anyway I can leave the original code on the processor-route even if it will no longer be used as the event will always have the uuid when reach the processor-route

Reply:
That code needs to move out of the debezium route and move it to the processor route.

Reply:
I think there is something you're missing in all this

Reply:
Sure. But I think it is not correct to look for UUI for records read from "reader-route" because the records can be deleted just after you put them on the "staging table". So I insist that we must look for UUID on the "debezium-route".